linux/power_operation_scripts/set_power_script.yml

```diff
@@ -78,33 +82,64 @@
 - name: "Update inventory"
   include_tasks: ../../common/update_inventory.yml
 
-- name: "Check the script file running as expected"
-  ansible.builtin.stat:
-    path: "{{ script_tag_path }}"
+- name: "Wait for VMware Tools running"
+  include_tasks: ../../common/vm_wait_vmtools_status.yml
+  vars:
+    vm_wait_vmtools_running: true
```

Comment:
For VM with VMTools installed, update_inventory.yml should have waited tools running. Please check your test logs. If it already did, this task can be removed. Or you can move this task before `update_inventory.yml` to make sure `update_inventory.yml` getting ip from vmtools